content/en/docs/tasks/observability/gateways/index.md

```diff
@@ -33,21 +33,21 @@ addons to require authentication when exposing them externally.
 
 1. [Install cert-manager](https://docs.cert-manager.io/en/latest/getting-started/install/kubernetes.html) to manage certificates automatically.
 
-1. [Install Istio](/docs/setup) in your cluster and enable the `cert-manager` flag and configure `istio-ingressgateway` to use
+1. [Install Istio](/docs/setup/install/operator) in your cluster and enable the `cert-manager` flag and configure `istio-ingressgateway` to use
```

Comment:
certmanager is deprecated.  I'd recommend sorting out (as a separate PR) how to communicate this.

Reply:
TODO followup PR to remove certmanager from the documentation.  It was never well integrated and currently is busted.